target/linux/ramips/dts/mt7620a_netgear_ex6100-v1.dts

```diff
@@ -0,0 +1,98 @@
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6100-v1", "ralink,mt7620a-soc";
+	model = "Netgear EX6100 v1";
+
+	aliases {
+		led-boot = &led_power_amber;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_amber;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_g {
+			label = "ex6100-v1:green:power";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_amber: power_a {
+			label = "ex6100-v1:amber:power";
+			gpios = <&gpio2 28 GPIO_ACTIVE_LOW>;
+		};
+
+		router_g {
+			label = "ex6100-v1:green:router";
+			gpios = <&gpio2 25 GPIO_ACTIVE_LOW>;
+		};
+
+		router_r {
+			label = "ex6100-v1:red:router";
+			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
+		};
+
+		router_b {
+			label = "ex6100-v1:blue:router-arrow";
+			gpios = <&gpio2 29 GPIO_ACTIVE_LOW>;
+		};
+
+		device_g {
+			label = "ex6100-v1:green:device";
+			gpios = <&gpio2 20 GPIO_ACTIVE_LOW>;
+		};
+
+		device_r {
+			label = "ex6100-v1:red:device";
+			gpios = <&gpio2 21 GPIO_ACTIVE_LOW>;
+		};
+
+		device_b {
+			label = "ex6100-v1:blue:device-arrow";
+			gpios = <&gpio2 22 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "ex6100-v1:green:wps";
+			gpios = <&gpio2 27 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		mode {
```

Comment:
No reset button?

Reply:
Defined in mt7620a_netgear_ex3700_ex6130.dtsi 
https://github.com/openwrt/openwrt/blob/2f1cc5c3d5e35d6aa76e794e3d5b4f5856cd38bc/target/linux/ramips/dts/mt7620a_netgear_ex3700_ex6130.dtsi#L14-L18